docs/setup/install/auto-enroll.asciidoc

```diff
@@ -0,0 +1,15 @@
+[role="exclude"]
+
+If this is the first time you're starting {kib}, this command generates a 
+unique link to enroll your {kib} instance with {es}.
+
+. In your terminal, click the generated link to open {kib} in your browser.
+
+. In your browser, paste the enrollment token for {kib} and click the
```

Comment:
Maybe it's worth explaining where to find the enrollment token? (e.g. "paste the enrollment token from {es} terminal and click the") 

Elasticsearch only logs an enrollment token for Kibana so we might not need to specifically mention that.

Reply:
Good thoughts @thomheymann. I think we can change this to:

>In your browser, paste the enrollment token that was generated in the terminal when you started {es}, and then click the button to connect your {kib} instance with {es}.